manager/controllers/app/modules/moduleinterface.go

```diff
@@ -17,22 +17,30 @@ type Transformations struct {
 	Allowed            bool
 	EnforcementActions []pb.EnforcementAction
 	Message            string
+	// geography relevant for the read/write operation
+	// indicates where the workload runs (for read access) or where to write the data to (for copying data to another location)
+	Geo string
 }
```

Comment:
Why is this in the Transformations?  This is contextual information that was one of the parameters used to make the decision as to whether to do the transformation, and while it's true that the manager will have to decide where to run the transformation ... I'm not sure why Geo should be part of the Transformations.

Reply:
There is no meaning to the Geo outside the operation context. This structure is used for communication with policy manager. As we discussed with @rohithdv, there is a single geography field passed to the connector - workload geography for the read, and the destination for the write. In the ingest flow we don't have a workload geography at all...

Reply:
Transformations name was misleading. Renamed to Operations. The idea is that the controller collects in advance governance actions for possible read and write operations, and selects modules later based on this information.